data/battle_scripts_1.s

```diff
@@ -6460,6 +6460,8 @@ BattleScript_SeedSowerActivates::
 	return
 
 BattleScript_AngerShellActivates::
+	saveattacker
```

Comment:
it is possible attacker wont be restored because it can jump to the `ButItFailed` script.

Reply:
Whoops, I missed that. Would it make more sense to have a `BattleScript_RestoreAttackerButItFailed` script that it jumps to? Or just move the popup to `BattleScript_AngerShellTryDef`

Reply:
I think `BattleScript_RestoreAttackerButItFailed` might become useful in the future so I would say this.

Though can you give an example of when it could cause an issue in a double batttle? Because if Anger Shell can cause an issue it might affect other stuff as well.

Reply:
but also why not just use a scripting battler?